### PR TITLE
Added description of generate command to the README

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

File: infra/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Continuous Integration infrastracture:
 
 | Command | Description |
 |---------|-------------
-| `generate`      | 
+| `generate`      | Generates skeleton files for a new target |
```
Comment (Dor1s) [inline infra/README.md]: Would you mind adding description for other commands too?

**Comment by alex** [comment]
Attempted to document the other commands, I'm 90% sure the descriptions are correct, with the exception of 1 TODO for a command I didn't really understand.

File: infra/README.md [hunk 2]
```diff
@@ -18,9 +18,9 @@ Continuous Integration infrastracture:
...
 | Command | Description |
 |---------|-------------
-| `generate`      | 
-| `build_image`   |
-| `build_fuzzers` |
-| `run_fuzzer`    |
-| `coverage`      |
-| `shell`         |
+| `generate`      | Generates skeleton files for a new target |
+| `build_image`   | Builds a docker image for a given fuzzer |
```
Comment (Dor1s) [inline infra/README.md]: "... for a given target"

File: infra/README.md [hunk 3]
```diff
@@ -18,9 +18,9 @@ Continuous Integration infrastracture:
...
+| `build_fuzzers` | TODO |
```
Comment (Dor1s) [inline infra/README.md]: Builds fuzzers for a given target

File: infra/README.md [hunk 4]
```diff
@@ -18,9 +18,9 @@ Continuous Integration infrastracture:
...
+| `run_fuzzer`    | Runs a fuzzer in a docker container |
+| `coverage`      | Runs a fuzzer in a docker container and computes a coverage report |
+| `shell`         | Starts a shell inside the docker image for a fuzzer |
```
Comment (Dor1s) [inline infra/README.md]: "... for a target"

**Comment by alex** [comment]
Done.

File: infra/README.md (hunk 3 above)
Comment (mikea) [inline infra/README.md]: builds all fuzzers into local directory

**Comment by mikea** [comment]
thanks

**Pull request merged**